chef/cookbooks/apache2/recipes/mod_ssl.rb

```diff
@@ -40,7 +40,8 @@
   # override the resource defined in default.rb; we don't want to create the
   # resource again, otherwise we will write the file twice
   resource = resources(template: "#{node[:apache][:dir]}/ports.conf")
-  resource.variables({apache_listen_ports: [node[:apache][:listen_ports], "443"].flatten})
+  node[:apache][:listen_ports] << "443"
+  resource.variables({apache_listen_ports: [node[:apache][:listen_ports]})
```

Comment:
Remove the extra `[` `]` around `node[:apache][:listen_ports]`, it's already a list

Reply:
(Wow, I didn't even close the needless brackets properly! But it's now fixed.)